.github/workflows/python-app.yml

```diff
@@ -12,7 +12,7 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
Why do we have to specify this? :thinking:

Reply:
ubuntu-latest is now 20.04 and there python 3.6 is not shipped. When running apt-get install python 3.6 it does not put the executable in the expected /usr/bin folder :shrug:

Reply:
But it should still be on PATH right? It's probably being installed in usr/local but that is ok

Reply:
Yes, it ends up somewhere. This is just minimal change for the bug-fix.